scripts/rust_is_available.sh

```diff
@@ -21,14 +21,20 @@ get_canonical_version()
 	echo $((100000 * $1 + 100 * $2 + $3))
 }
 
+_show_documentation_on_exit() {
```

Comment:
The leading underscore is intended to mean internal, right?

I see only some scripts following the convention in the kernel, all in `tools/` (except 1 for a trap function) and of those some seem to use it like in C to have a "lower-level"/raw function with the same name.

Personally, I like the convention (visibility/privacy is good), but since we are not following in the file (I don't think `get_canonical_version` should be used from outside either?), so I think we can skip it here, and if we want to follow it, we can do so in another patch.

Reply:
ok thanks for the suggestion. yes, the intention was to mean internal for better readability